qmlti/model.php

```diff
@@ -0,0 +1,122 @@
+<?php
```

Comment:
Name the file after the class... If you have a name collision its because you need some folders to organise your files ;-)